_assignments/mp3.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: assignment
+title: "MP3: VR Critic"
+due: "March 4, 2019 @ 11:59PM"
+material: 
+  - name: Oculus Best Practices Guide
+    url: https://developer.oculus.com/design/latest/concepts/book-bp/
+points: 100
+notes: false
+instructions: false
+---
+
+You will review three VR experiences according to the [Oculus Best Practices Guide](https://developer.oculus.com/design/latest/concepts/book-bp/). These VR experiences can be from the Oculus Store, the Steam Store, or a student demo from the CS498 `V:` drive (in the "Student Projects" folder). There is no right or wrong answer. This assignment is meant to turn you into a VR snob; pick apart every little design decision you can think of.
+
+Each review must focus on the following four categories:
+1. Graphical/Aesthetic Quality
+1. User interface
+1. Movement
+1. Immersion/Presence
+
+For each category, write a short paragraph that describes what that particular VR experience does well and what it does poorly in regards to that category. Include at least two suggestions on how to improve the experience. Make sure to back up your critiques and suggestions with evidence from the Best Practices Guide. In total, write 150-300 words per experience.
+
+#### Grading:
+
+*You will be graded according to how well you can justify your answer with specific examples from the Oculus Best Practices Guide.*
+
+- Each review must cover all four categories. 
```

Comment:
this and "must include at least two suggested improvements" is repeated above

Reply:
It's like a rubric -- the assignment is introduced in the first section, and what students are graded on is listed in the rubric, except we don't attach point values to the grading criteria here. Given that, I think the repetition is fine, but I'm open to suggestions to reduce the length of the assignment.